conditional: Document the bootvar0 variable (#4183)
raspberrypi/documentation: documentation/asciidoc/computers/config_txt/conditional.adoc

----
The value of the `boot_arg1` variable when the OS was started can be read via xref:configuration.adoc#part4[device-tree] at `/proc/device-tree/chosen/bootloader/arg1`

==== `bootvar0`
Raspberry Pi 5 and newer devices only.

The `bootvar0` variable is a 32-bit user defined value which is set through rpi-eeprom-config, and then can be used as a conditional variable in config.txt.

For example, setting `bootvar0` to 42 via `rpi-eeprom-config`:
[source,ini]
----
BOOTVAR0=42
----

and then using it conditionally in `config.txt`:
[source,ini]
----
[bootvar0=42]
arm_freq=1000
----

> This allows a common image (i.e. with the same config.txt file) to support different configurations based on the persistent rpi-eeprom-config settings.

Reviewer: ```suggestion
This allows a common image (that is, with the same `config.txt` file) to support different configurations based on the persistent `rpi-eeprom-config` settings.
```
